pkg/cmd/dev/bench.go

```diff
@@ -18,8 +18,9 @@ import (
 )
 
 const (
-	benchTimeFlag = "bench-time"
-	benchMemFlag  = "bench-mem"
+	benchTimeFlag  = "bench-time"
+	benchMemFlag   = "bench-mem"
+	crdbTestOnFlag = "crdb-test-on"
```

Comment:
Drop the "-on", this is already a boolean flag so the "on" feels implied. I would actually prefer not providing this as a flag in `dev` at all since it's unlikely that it should ever be used (`make` didn't), and if it is, you can through `dev bench -- --crdb_test --test_env=COCKROACH_INTERNAL_DISABLE_METAMORPHIC_TESTING=true`. If folks find a need for it, or notice a regression, we can add this flag then.

Reply:
So if bazel sees --crdb_test_off followed by --crdb_test last one wins?  Cool, I'll remove it.